torch/utils/tensorboard/writer.py

```diff
@@ -837,6 +837,26 @@ def add_custom_scalars(self, layout):
         """
         self._get_file_writer().add_summary(custom_scalars(layout))
 
+    def add_mesh(self, tag, vertices, colors=None, faces=None, config_dict=None, global_step=None, walltime=None):
+        """Add mesh or point cloud data to summary.
```

Comment:
Can you add a paragraph below this giving more details on the method? @lanpa feel free to add here too.